pkg/gogocodec/codec_test.go

```diff
@@ -11,7 +11,6 @@
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
 // limitations under the License.
-
```

Comment:
what's with the changes in this file?

Reply:
I was resolving conflicts but forgot that I needed to remove this change.